src/js/types/ProcedureLeg/AFLeg.ts

```diff
@@ -0,0 +1,19 @@
+import { LegType, ProcedureLegBase, TurnDirection } from "."
+import { Fix } from "../fix"
+import { Degrees, NauticalMiles } from "../types"
+
+export interface AFLegData extends ProcedureLegBase {

```

Comment:
Why the extra whitespace?

Perhaps a preparation for documentation? That'd be great to have on the TS side as well, with the same issue of maintainability that has already been mentioned...

Reply:
If you're refering to the space between, these are the same typings i've writtern for other projects so I guess it was just there already, perhaps we need an eslint rule for that

Reply:
Ah, I see!